examples/webhook.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/xanzy/go-gitlab"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+// parse errors
```

Comment:
In Go, comments are suppose to be sentences. So they should start with a capital letter ann end with a dot.

Reply:
Please end your comments with a dot...